src/connect/planner.cpp

```diff
@@ -59,7 +59,8 @@ namespace {
 
     bool path_allowed(const char *path) {
         constexpr const char *const usb = "/usb/";
-        const bool is_on_usb = strncmp(path, usb, strlen(usb)) == 0;
+        // Note: allow even "bare" /usb
+        const bool is_on_usb = strncmp(path, usb, strlen(usb)) == 0 || strcmp(path, "/usb") == 0;
```

Comment:
:thinking: I don't like the 2 separate calles to strcmp but at this moment I don't see a simple way of elliminating one or the other

Reply:
I don't see an elegant one either. Writing some kind of search automaton is obviously an overkill and the compiler is likely doing that under the hood (I wouldn't be surprised if it does one 4-byte int compare and then checking for either `/` or `\0` or something like that).